controllers/AuthController.js

```diff
@@ -110,11 +110,9 @@ export default class AuthController {
 
     const user = await User.findOne({ where: { email } });
 
-    if (!user) {
-      return res.status(401).send({
-        message: 'You Entered an incorrect Email or Password',
-      });
-    }
+    if (!user) errorResponse(res, 401, { message: 'You Entered an incorrect Email or Password' });
+    if (user.status === 'inactive') errorResponse(res, 401, { message: 'You Have been banned, Please contact an admin' });
```

Comment:
there's no return statement here, this would cause the headers to be sent twice

Reply:
okay I'll fix that, thank you